theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/Dot.tsx

```diff
@@ -75,7 +75,7 @@ const HitZone = styled.div`
   }
 
   &:hover + ${Square}, &.beingDragged + ${Square} {
-    ${dims(dotSize + 5)}
+    ${dims(DOT_SIZE + 5)}
```

Comment:
Question: What is `+ 5`?
This kinda looks like something else that should be a constant with a name.

Reply:
Added a constant based on a brief look at the code